src/wallet/test/init_test_fixture.cpp

```diff
@@ -10,7 +10,7 @@
 
 InitWalletDirTestingSetup::InitWalletDirTestingSetup(const std::string& chainName) : BasicTestingSetup(chainName)
 {
-    m_wallet_client = MakeWalletClient(*m_chain, *Assert(m_node.args));
+    m_wallet_client = MakeWalletClient(m_node, *m_chain, *Assert(m_node.args));
```

Comment:
Shouldn't it be an address to the m_node that needs to be passed here?

Reply:
So it's passing a reference (`m_node` is already a reference). I could make it into a pointer, but I think this works? Let me know if I'm doing something daft without realising it.

Reply:
I guess this works. We can leave it here for now.